### PR TITLE
feature/sign-in

### PR DESCRIPTION
**Pull request by HO-LE-ANH-TOAN**
comfirm voi

## Summary by CodeRabbit

- **New Features**
	- Introduced a new ForgotPassword component for streamlined password recovery.
	- Added a SignUp component for user registration with validation and social media login options.
	- Enhanced the existing SignIn component with improved validation and animations.
	- Implemented Google authentication functionality through a new Test component.
  
- **Styling Enhancements**
	- Implemented custom fonts, 'Air America' and 'Air Americana,' to improve the application's visual aesthetics.

- **Routing Updates**
	- Expanded routing to include new paths for SignIn, SignUp, ForgotPassword, and Test components for better user management.

- **Dependencies**
	- Added new libraries for enhanced UI components and animations: `@mantine/core`, `@mantine/hooks`, `framer-motion`, `react-facebook-login`, `react-github-login`, `react-oauth`, and `@react-oauth/google`.

### CODE REVIEW

**Review by catchmeifyoucan99 — DISMISSED** [review]
Never Give Up

File: Client/src/modules/core/components/account/SignIn.jsx [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+                src="https://via.placeholder.com/40x40"
+                alt="Icon 3"
+                className="w-10 h-10"
+              />
+            </motion.div>
+          </form>
+        </div>
+      </div>
+    </div>
+  );
```
[the remark this answers is not in the document]
Comment (htilssu) [reply Client/src/modules/core/components/account/SignIn.jsx]: @catchmeifyoucan99

**Comment by htilssu** [comment]
@coderabbitai review

**Comment by htilssu** [comment]
@coderabbitai review all

File: Client/src/pages/sign-in/SignIn.jsx [hunk 2]
```diff
@@ -0,0 +1,389 @@
...
+  //       }
+  //   },
+  //   "id": "",
+  //   "userID": "",
+  //   "expiresIn": ,
+  //   "accessToken": "",
+  //   "signedRequest": ".eyJ1c2VyX2lkIjoiMTM3MzU2OTE4NDA0OTQxOCIsImNvZGUiOiJBUUJJbTQxbjc0ZUloVlVDVXltZVhPb3A1LWZ1cnpDLUhOMGdRWm9STEhmVF90UG1DNG1WSWpPRHhXUE9YOEpkNmkyd1J5YzJxdFBfSDhuOXk5dTJ6YVlkR215NTFtSVJNUk5MMm1LenBQS3A0ZVdnYXRHY2lZS1BrY0UtUElHNUZMZG84T3FyMXJwdVJCT0haN24zNXJKX0lBR0oyTHdydV9kdmRfME9HNERZMTdSV1pqeFJHaHlHQkd2WVJ6UTBIX0xkWTFuMkJQdmowclluWENONFJTU3ZMdERJU3JyNS1ZRnN4U05WaWdMRjF3cE4wN3p5dDJSV3FyRkk1SWt0dVZiT1l3S01DRHM0enFYNWxzNkNiN1VZelpiQ3lnNDAtRzU3d0hENExTekNtOWdFVG05RHZjc0J6OWJlMXNFVkkycFVfZURsd3BvSVFRUmhYNkdSTE1pSnVmTkV4ZHpjRVc0THU5RWhxZ2x2WGciLCJhbGdvcml0aG0iOiJITUFDLVNIQTI1NiIsImlzc3VlZF9hdCI6MTcyNjYzODkzMn0",
+  //   "graphDomain": "facebook",
+  //   "data_access_expiration_time": 
+  // }
```
Comment (htilssu) [inline Client/src/pages/sign-in/SignIn.jsx:72]: lộ nè fix nha

**Review by htilssu — CHANGES_REQUESTED** [review]

**Pull request closed without merging**